lib/tasks/sample_data/payment_method_factory.rb

```diff
@@ -36,16 +35,6 @@ def create_cash_method(enterprise)
       )
     end
 
-    def create_card_method(enterprise)
-      create_payment_method(
-        Spree::Gateway::Bogus,
```

Comment:
Yeah, as @filipefurtad0 mentioned `create_card_method` didn't work before, so I'm just removing malfunctioning code. The check method does work, though.

I could add the Paypal gateway, but I'm not sure if we can set it to test mode, so that users don't accidently make a charge.

I think the Stripe gateway is a no go because users need an API key to register it. I guess we can fake one though. I don't know enough about the gateways to know which one is the better choice, that is why I mentioned it in the initial comment.